### DISCUSSION TITLE
Duplicating map in Unreal Engine

### DISCUSSION THREAD

**Discussion by a-yildiz** [carla-simulator/carla#4783] [Q&A]
Hello,
I have Carla 0.9.12 compiled from the source.

What I'd like to do is very simple: I want to duplicate the `Town01_Opt` map and replace all the traffic lights with stop signs. Let's call this map `Town01_Opt_mod`. I managed to replace the objects in Unreal Engine and save it as a new map under the name `Town01_Opt_mod`.

But I get an the following error when I try to load this map using the PythonAPI:
`ERROR: unable to parse the OpenDRIVE XML string.`

What should I do?

Thanks.

**Comment by chrninja** [comment]
@a-yildiz How did you go about doing this? I want to add some parked vehicles to `Town01_Opt` but not necessarily overwrite the working `Town01_Opt`

**Comment by ajperry2** [comment]
This worked for me. Sorry if I'm late.

- Go into Carla/Maps/OpenDrive
- Duplicate the related Default Map's xodr file.
- Rename your copied xodr file to have the same name as your map.
